libs/designsystem/src/lib/components/page/page-footer/page-footer.component.html

```diff
@@ -0,0 +1 @@
+<ng-content></ng-content>
```

Comment:
Footer should wrap content in a layout container width max-width:
```suggestion
<div class="wrapper">
  <ng-content></ng-content>
</div>
```
And in scss:
```css
.wrapper {
  position: relative;
  max-width: var(--page-content-max-width, 720px);
  margin: 0 auto;
}
```